code/modules/response_team/ert_outfits.dm

```diff
@@ -37,6 +37,7 @@
 	rt_mob_job = "ERT Commander"
 
 	uniform = /obj/item/clothing/under/rank/centcom_officer/sensor
+	gloves = /obj/item/clothing/gloves/combat
```

Comment:
Why not just put this on the `/datum/outfit/job/centcom/response_team` level?

Reply:
Mainly cause I did not think of it, since that datum was in it's separate file, but it is done now.